pajbot/apiwrappers/twitch/helix.py

```diff
@@ -602,3 +602,17 @@ def get_channel_emotes(
             expiry=60 * 60,
             force_fetch=force_fetch,
         )
+
+    def send_chat_announcement(self, bot_id: str, body: Dict[str, str], authorization, channel_id: str = None):
```

Comment:
It would be nice to have a docstring here for the method and its parameters. Documentation should state things like: an exeption is raised if something is not successful, `sender_id` must be the same user as the user authenticated by the given `authorization`, and that the message should (must? Please test the APIs behaviour on this) be cropped before handing it to this function